mindmeld/converter/rasa_config.txt

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""This module contains a template MindMeld app configuration"""
+
+# The namespace of the application. Used to prevent collisions in supporting services across
+# applications. If not set here, the app's enclosing directory name is used.
+# APP_NAMESPACE = 'app-name'
+
+# Dictionaries for the various NLP classifier configurations
+
+# An example decision tree model for intent classification
+INTENT_CLASSIFIER_CONFIG = {
```

Comment:
remove this intent classifier

Reply:
Do you still want me to remove this?

Reply:
Ah, sorry, my bad. I didn't want to use the default intent config.

Reply:
Ok, I changed it to the ENTITY_RECOGNIZER_CONFIG you gave me.